### ISSUE TITLE
Simple transport mode prediction

### ISSUE THREAD

**Issue by henrymartin1** [mie-lab/trackintel#36]
This functionality should be based on properties of the movement data. A robust classification could be {slow, medium, fast}.

**Comment by julienneross** [comment]
Does any of you have or a link to any data prediction that is already program? I would love to study it. Thanks!

**Comment by henrymartin1** [comment]
Hi @julienneross, 
Nice to hear that you are interested in this! The implementaion of a generalizable transport mode prediction in our library is a complex task because it often depends strongly on the dataset you are working with. For now we are planning on implementing a simplified transport mode prediction that predicts aggregated modes e.g., unmotorized vs. slow motorized vs. fast motorized. 
A big issue for training transport mode classifiers is always the availablility of datasets with ground truth labels. I think the [Geolife ](https://www.microsoft.com/en-us/download/details.aspx?id=52367) dataset has some labels. You can already use trackintel to import and aggregate the Geolife dataset (without labels) with trackintel. This is shown in 
trackintel/examples/example_geolife/Trackintel_introduction.ipynb

Best,
Henry

**Comment by dominikbucher** [comment]
I wrote functions for a simple identification as:

```
if speed < 15 / 3.6:
    return 'slow_mobility'
elif speed < 100 / 3.6:
    return 'motorized_mobility'
else:
    return 'fast_mobility'
```

The idea would be to learn the boundaries (15 km/h and 100 km/h) on one of our datasets and hardcode it here. @henrymartin1 you had something like this in mind too?

**Comment by henrymartin1** [comment]
I think for the beginning there is not much more that we can do than that. I think _learning_ the boundaries based on our datasets is a bit much I would rather say that we could double check it with our datasets...

A goal for the bachelor thesis could be to provide some pretrained models (as discussed I think...).

**Comment by dominikbucher** [comment]
Yeah I mean we could learn the optimal boundaries, but I'm also not so sure how much (given that this is a very crude classification anyways) it would help. I'll do the pull request like this for now, and then we'll adapt this method and add others on top (as part of a BSc/MSc thesis).

**Comment by henrymartin1** [comment]
This is PR #65
